powerline/segments/vim/plugin/coc.py

```diff
@@ -0,0 +1,47 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+	import vim
+except ImportError:
+	vim = object()
+
+from powerline.bindings.vim import vim_command_exists
+from powerline.theme import requires_segment_info
+
+# coc_status's format: E1 W2
+def parse_coc_status(coc_status):
+	# type(coc_status) is tuple
+	errors_count = 0
+	warnings_count = 0
+	if len(coc_status) <= 0:
+		return errors_count, warnings_count
+	status_str = coc_status[0]
+	if len(status_str) <= 0:
+		return errors_count, warnings_count
+	status_list = status_str.split(' ')
+	for item in status_list:
+		if len(item) > 0 and item[0] == 'E':
+			errors_count = int(item[1:])
+		if len(item) > 0 and item[0] == 'W':
+			warnings_count = int(item[1:])
+	return errors_count, warnings_count
+
+@requires_segment_info
+def coc(segment_info, pl):
+	segments = []
```

Comment:
The documentation for this function is missing: `powerline-lint` does not know how the highlight groups are named. Try adding:
```
    '''<add description here>

    Highlight groups used: ``coc:warning`` or ``warning``, ``coc:error`` or ``error``.
    '''
```

Reply:
Thanks for the tip.